client/src/components/pages/landing/footer/footer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './footer.module.scss'
+
+interface IFooterProps {}
+export const Footer = (props: IFooterProps) => {
```

Comment:
Why do you need empty requisites? 
Maybe then it makes sense to take an array of names and references, in case of changes in the project

Reply:
Well, we have a container for this page if in the future we need to add some logic here (for example some redux tricks), but for now, it is just a static landing page. So if we add logic we should just change props interfaces and it's done.

I'd like to leave it as it is

Reply:
Okay